lib/stack_master/template_compiler.rb

```diff
@@ -16,12 +16,8 @@ def self.register(name, klass)
     end
 
     # private
-    def self.template_compiler_for_stack(template_file_path, sparkle_pack_template, config)
-      ext = if sparkle_pack_template
-        :rb
-      else
-        file_ext(template_file_path)
-      end
+    def self.template_compiler_for_stack(template_file_path, config)
+      ext = file_ext(template_file_path)
       compiler_name = config.template_compilers.fetch(ext)
```

Comment:
It's necessary to specify `compiler` for sparkle pack templates. Using symbol to lookup because it's hard to lookup the template by file name, and there could technically be multiple templates defined in a single file.